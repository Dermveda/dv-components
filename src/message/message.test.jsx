import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import Message from './message.jsx';
const props = {
	message: 'Test message'
};
describe('<Message/>', () => {
	it('Will render', () => {
		const el = renderer.create(<Message {...props} />);
		expect(el).toMatchSnapshot();
	});
	it('Will not have message', () => {
		const el = shallow(<Message message="" />);
		expect(el).toMatchSnapshot();
	});
	it('Will be success', () => {
		const el = shallow(<Message {...props} isSuccess={true} />);
		expect(el).toMatchSnapshot();
	});
	it('Will be info', () => {
		const el = shallow(<Message {...props} isInfo={true} />);
		expect(el).toMatchSnapshot();
	});
});
