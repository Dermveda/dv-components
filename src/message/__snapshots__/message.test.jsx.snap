// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Message/> Will be info 1`] = `
<div>
  <div
    className="mt-2 alert alert-info"
    role="alert"
  >
    Test message
  </div>
</div>
`;

exports[`<Message/> Will be success 1`] = `
<div>
  <div
    className="mt-2 alert alert-success"
    role="alert"
  >
    <strong>
      Awesome!
       
    </strong>
    Test message
  </div>
</div>
`;

exports[`<Message/> Will not have message 1`] = `<div />`;

exports[`<Message/> Will render 1`] = `
<div>
  <div
    className="mt-2 alert alert-danger"
    role="alert"
  >
    <strong>
      Oops!
       
    </strong>
    Test message
  </div>
</div>
`;
